extras/Studio.framer/framer/version

```diff
@@ -1 +1 @@
-7
\ No newline at end of file
+8
```

Comment:
This is just the template version, so the Studio.framer project can be used with the beta. Does not really belong in this PR, sorry.